resources/variants/cartesio_0.25.inst.cfg

```diff
@@ -4,9 +4,8 @@ version = 2
 definition = cartesio
 
 [metadata]
-author = Cartesio
```

Comment:
Removed the author?

Reply:
This is the same for all variants, we have removed the `author` field because it's not used anywhere. If for the 3rd-party machines, this field is sensitive, we can put them back. @Ghostkeeper @jackha

Reply:
Added authors back to 3rd-party variants in 072c85bac